frontend/projects/valtimo/plugin/src/lib/plugins/zaken-api/components/create-zaak/create-zaak-configuration.component.scss

```
@@ -33,35 +24,11 @@ button.delete-button:hover {
-div.property-row {
-  width: 100%;
-  display: grid;
-  grid-template-columns: 92% 8%; /* precise 95/5 split */
-}
-div.property-row > * {
-  box-sizing: border-box; /* keeps borders/padding inside the column width */
-  min-width: 0; /* prevents overflow from long content */
-}
-div.property-delete-col {
-  margin-left: auto;
+:host ::ng-deep .add-button .v-overflow-menu__pane {
```

Stylelint (17.9.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)